doc/operator-user-guide.md

```diff
@@ -711,6 +713,61 @@ spec:
 Only when the underlying PersistentVolume's storageclass allows resizing, storage resource requirements can be modified after installation.
 Check [Expanding persistent volumes](https://docs.openshift.com/container-platform/4.5/storage/expanding-persistent-volumes.html) official doc for more information.
 
+#### Setting custom PriorityClassName
+PriorityClassName specifies the Pod priority.  
+It be can be customized through APIManager CR `priorityClassName` attribute for each DeploymentConfig.  
+Example for apicast-staging and backend-listener:
+```yaml
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIManager
+metadata:
+    name: example-apimanager
+spec:
+    wildcardDomain: api.vmogilev01.0nno.s1.devshift.org
```

Comment:
wildcard domain with `example.com` value

Reply:
done. Thank you